assets/app/scripts/controllers/build.js

```diff
@@ -99,8 +99,34 @@ angular.module('openshiftConsole')
           if (!$filter('isIncompleteBuild')(build)){
             delete $scope.buildConfigBuildsInProgress[buildConfigName][buildName];
           }
-        }        
+        }
       }));
+
+      (function initLogs() {
```

Comment:
Updated, will only flip the bool if the `onerror` handler is called on the websocket.